Nodejs/Product/TestAdapter/TestFrameworks/Angular/vsKarmaReporter.js

```diff
@@ -0,0 +1,116 @@
+﻿// @ts-check
+
+const http = require('http');
+const path = require('path');
+
+const jasmineReporterPath = path.resolve(process.env.VSTESTADAPTERPATH, 'jasmineReporter.js');
+const testCases = JSON.parse(process.env.TESTCASES);
+const isDiscovery = process.env.ISDISCOVERY === 'true';
+
+const vsKarmaReporter = function (baseReporterDecorator, config, logger, emitter) {
```

Comment:
Any particular reason you chose to write this as an ES5 class?
I feel like it would be much more readable if we could just use ES6 classes.

Reply:
You mean `= function` the `$inject` and `export`? If that's the case, I just followed the documentation on creating Karma reporters. I didn't saw any example that didn't use this so I just rolled with it. But I do agree with you that having an ES6 class is nicer to eye. 

Let's do a quick call on this.